web/styles/popovers.css

```diff
@@ -1135,7 +1135,7 @@ ul.popover-group-menu-member-list {
 
     .full-name {
         /* 18px at 14px/1em */
-        font-size: 1.2857;
+        font-size: 1.2em;
```

Comment:
Reason for change: `.tippy-content` is at 15px. There's an error in the comment, will update soon to `/* 18px at 15px/1em */`